digid_eherkenning/mock/idp/digid_urls.py

```diff
@@ -1,13 +1,13 @@
 from django.urls import path
 
 from digid_eherkenning.mock.idp.views.digid import (
+    DigiDMockIDPBSNLoginView,
     DigiDMockIDPLoginView,
-    DigiDMockIDPPasswordLoginView,
 )
 
 app_name = "digid-mock"
 
 urlpatterns = [
     path("inloggen/", DigiDMockIDPLoginView.as_view(), name="login"),
-    path("inloggen_ww/", DigiDMockIDPPasswordLoginView.as_view(), name="password"),
+    path("inloggen_ww/", DigiDMockIDPBSNLoginView.as_view(), name="bsn"),
```

Comment:
```suggestion
    path("inloggen_bsn/", DigiDMockIDPBSNLoginView.as_view(), name="bsn"),
```
`inloggen_ww` is probably mnemonic for `inloggen_wachtwoord` (at least that's what comes to my mind), which is no longer accurate.